src/blocklypropclient.js

```diff
@@ -82,15 +71,15 @@ var client_domain_port = 6009;
  *
  * @type {string}
  */
```

Comment:
Bump minimum version to remove the need to support non-base64 encoded streaming.
Will show a warning for anyone using versions below the recommended version (but they will still work).